tests/bootable-tests/src/test/resources/test-project/package-bootable-grpc-pom.xml

```diff
@@ -0,0 +1,60 @@
+<!--
+~ JBoss, Home of Professional Open Source.
+~
+~ Copyright 2018 Red Hat, Inc., and individual contributors
+~ as indicated by the @author tags.
+~
+~ Licensed under the Apache License, Version 2.0 (the "License");
+~ you may not use this file except in compliance with the License.
+~ You may obtain a copy of the License at
+~
+~     http://www.apache.org/licenses/LICENSE-2.0
+~
+~ Unless required by applicable law or agreed to in writing, software
+~ distributed under the License is distributed on an "AS IS" BASIS,
+~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~ See the License for the specific language governing permissions and
+~ limitations under the License.
+-->
```

Comment:
I'm sure this is just from a copy/paste, but we should use the new copyright header.